devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy

```diff
@@ -188,7 +189,7 @@ def configure() {
 	if (manufacturer == "Heiman"|| manufacturer == "HEIMAN") {
 		return refresh() +
 		        zigbee.temperatureConfig(30, 300) +
-		        zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021, DataType.UINT8, 30, 21600, 0x10) +
+		        zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021, DataType.UINT8, 30, 3600, 0x10) +
```

Comment:
A one minute reporting interval is too short. Why do you think it's necessary?

Reply:
I thought 3600 is an hour